components/elm/docs/tech-guide/shortwave_radiation.md

```diff
@@ -0,0 +1,807 @@
+# Overview
+
+The main features of ELM shortwave radiation model are as
+follows:
+
+- The shortwave radiation model in ELM computes the beam
+  (or direct) and diffuse radiation absorbed by the canopy
+  (i.e. sunlit and shaded leaves) and the ground, and
+  the radiation reflected to the atmosphere.
+- The model accounts for radiation in two bands: visible
+  ($< 0.7\mu$m) and near-infrared ($\ge 0.7 \mu$m).
+- The model uses the two-stream approximation of
+  Dickenson 1983[@dickinson1983land] and
+  Sellers 1985[@sellers1985canopy].
+- The model is a single-layer canopy model.
+- Sunlit leaf absorbs unscattered direct radiation,
+  scattered direct radiation, and diffuse radiation.
+- Shaded leaf absorbs scattered direct radiation and
+  diffuse radiation.
+- Snow albedo is modeled using the hybrid SNICAR-AD model
```

Comment:
"Snow albedo and the shortwave radiative heating of each snowpack layer is modeled..."